packages/builders/src/node/build/webpack/config.ts

```diff
@@ -0,0 +1,141 @@
+import * as webpack from 'webpack';
+import { Configuration, ProgressPlugin } from 'webpack';
+
+import * as ts from 'typescript';
+import { dirname, resolve } from 'path';
+
+import { LicenseWebpackPlugin } from 'license-webpack-plugin';
+import CircularDependencyPlugin = require('circular-dependency-plugin');
+import ForkTsCheckerWebpackPlugin = require('fork-ts-checker-webpack-plugin');
```

Comment:
Does the angular webpack config use the same plugin?

Reply:
Angular webpack config does not use the same plugin because it uses the AngularCompilerPlugin to run AoT compilation. I'm not certain, but theoretically they would be able to split type checking as well.